Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -1040,16 +1040,6 @@ WarpXParticleContainer::PushX (int lev, Real dt)
         {
             Real wt = amrex::second();
 
-            auto& attribs = pti.GetAttribs();
-
-            auto& uxp = attribs[PIdx::ux];
-            auto& uyp = attribs[PIdx::uy];
-            auto& uzp = attribs[PIdx::uz];
-
-            const long np = pti.numParticles();
-
-            giv.resize(np);
```

Comment:
No need to allocate `giv` anymore: this is now computed on the fly, inside the kernel.